src/base/xml.cpp

```diff
@@ -772,6 +772,11 @@ void XML_Node::build(const std::string& filename)
 
 void XML_Node::build(std::istream& f, const std::string& filename)
 {
+    warn_deprecated("XML_Node::build",
+        "\nThe CTI and XML input file formats are deprecated and will be removed in\n"
+        "Cantera 3.0. Use `cti2yaml.py` or `ctml2yaml.py` to convert CTI or XML input\n"
+        "files to the YAML format. See https://cantera.org/tutorials/legacy2yaml.html\n"
+        "for more information.");
```

Comment:
Should these be regular single quotes rather than back-ticks? (Here and elsewhere)

Reply:
On a separate note, would it make sense to promote the `legacy2yaml` page to a more prominent position on the tutorials page (right now it's within the input-files sub-category)?

Reply:
I'm starting to wondering if a large chunk of the "Input" documentation needs to be promoted a level in the website -- Right now, it's all buried as a subsection under "tutorials", which has the effect of both putting the input file related tutorials and extra layer deep, and hiding the YAML format documentation inside the tutorials section, which isn't really correct.

Reply:
Actually, I take that back - Besides being accessible from within the "Tutorials" section, the general YAML documentation is available from the main `cantera.org` page under Documentation -> YAML Input File Users' Guide, and the documentation for specific models is available as Documentation -> YAML Input File API Reference, which seems about right.

I'm not sure how much more prominent the `legacy2yaml` page needs to be. Within the tutorials section, I think it's in the right place, though I suppose it could come before the deprecated options of "creating a new CTI file" and "conversion from Chemkin to CTI". For users who don't think they need tutorials, an additional link on the "Documentation" page wouldn't hurt either.